source/getting-started/tutorials/sensors/index.md

```diff
@@ -0,0 +1,283 @@
+That means we have to satisfy these three inequalities:
+
+$$
+\frac{\rm PWM\ frequency}{\rm event\ ratio} \ge \rm TASK\_UPDATES\_PER\_SEC \\
+\\
+\frac{\rm event\ ratio}{\rm PWM\ frequency} \ge \rm Sensor\ Collection\ Time \\
+\\
+\frac{1}{\rm TASK\_UPDATES\_PER\_SEC} < \rm Control\ Task\ Time
+$$ (eq:vout)
+
+The first inequality is necessary to ensure the control task can run at the specified rate of `TASK_UPDATES_PER_SEC`.\
+The second inequality ensures that the sensors don't take up the entire timeslot.\
+The third inequality makes sure that we are able to run the control task in the allotted time slot.\
+These three combined inequalities give us both an upper and lower bound for User Event Ratio and PWM frequency relative to each other.
+
+## C-Code
+
+In this tutorial, we'll be activating the timing manager in Post-Sensor mode. Let's enable Post-sensor mode in the `user_config.h` file by setting `USER_CONFIG_ISR_SOURCE` to `1`
+
+`user_config.h`:
+```
+// Specify the source of the scheduler ISR
+// Mode 0: legacy mode - scheduler is triggered based on the PWM carrier events and ratio
+//         of carrier frequency to desired control frequency
+// Mode 1: post-sensor mode - scheduler is triggered when all the enabled sensors are done
+//         acquiring their data
+#define USER_CONFIG_ISR_SOURCE (1)
+```
+
+Let's enable one of the sensors to start observing the effects of the timing manager. In the `controller_init()` function, enable the ADC (analog to digital converter) with `timing_manager_enable_sensor(ADC)`.
+
+`app_controller.c`:
+```
+void app_controller_init(void)
+{
+    // Enable data sampling for ADC
+    timing_manager_enable_sensor(ADC);
+    // Register "ctrl" command with system
+    cmd_controller_register();
+}
+```
+
+To get data from the ADC, use the function `analog_getf(ANALOG_IN1, &output)`. Read about the analog channel mapping on the [analog input page](/hardware/subsystems/analog.md). We won't be using the ADC's actual numerical output in this tutorial, we're just enabling it to activate the timing manager.
+
+To understand the specific timings of sensor collection and tasks, we need to know the specific numbers of the factors that control tasks.
+ - User set `TASK_CONTROLLER_UPDATES_PER_SEC` is set in `task_controller.h`, it is (10000)
+ - PWM frequency can be set with a hardware command `hw pwm sw`, but the default value is in `common/drv/pwm.h` at (100000.0)
+ - User Event Ratio is set in `common/drv/timing_manager.c` in the `timing_manager_init()` function. It is set to `TM_DEFAULT_PWM_RATIO`, which is 10.
+ - Sensor collection time for the ADC can be gathered with the hardware command `hw tm time adc` or the C function `timing_manager_get_time_per_sensor(ADC)`. It is around 0.86 microseconds.
+ - The control task time can be gathered with the user-made command `ctrl stats print`. We are specifically looking at the Run-Time.
```

Comment:
More code formatting here:

`#define`d values should be enclosed in backticks.

"set in `file/path/file.c` _with_ the `timing_manager_init()` function."

"The ADC's data acquisition time can be retrieved with the..."

"The control task's run-time and loop-time can be retrieved with..."